<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputManagement</class>
 <widget class="QWidget" name="OutputManagement">
    <property name="geometry">
     <rect>
        <x>0</x>
        <y>0</y>
        <width>980</width>
        <height>680</height>
     </rect>
    </property>
    <property name="windowTitle">
     <string>OutputManagement</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#OutputManagement { background-color: #F4F6F8; }
QLabel { color: #2B2F33; }
QFrame#cardFrame, QFrame#detailsFrame { background-color: #FFFFFF; border: 1px solid #CBD5DF; border-radius: 12px; }
QLineEdit, QComboBox, QDateEdit { background-color: #FFFFFF; border: 1px solid #7F8B96; border-radius: 6px; padding: 4px 8px; }
QLineEdit::placeholder { color: #8A9099; }
QLineEdit:focus, QComboBox:focus, QDateEdit:focus { border: 1px solid #2F7D66; }
QComboBox::drop-down { border-left: 1px solid #7F8B96; width: 26px; }
QComboBox::down-arrow { image: url(:/new/prefix1/style/QSS_IMG/go-down-symbolic.symbolic.png); width: 12px; height: 12px; }
QComboBox::down-arrow:on { image: url(:/new/prefix1/style/QSS_IMG/go-up-symbolic.symbolic.png); }
QWidget#OutputManagement QComboBox QAbstractItemView { background-color: #FFFFFF; color: #2B2F33; selection-background-color: #DDEFE9; selection-color: #2B2F33; border: 1px solid #7F8B96; outline: 0; }
QWidget#OutputManagement QComboBox QAbstractItemView::item { background-color: #FFFFFF; color: #2B2F33; padding: 6px 10px; }
QWidget#OutputManagement QComboBox QAbstractItemView::item:hover { background-color: #EEF7F3; }
QWidget#OutputManagement QComboBox QAbstractItemView::item:selected { background-color: #DDEFE9; color: #1E2A24; }
QPushButton { background-color: #F7F9FB; color: #1F2328; border: 1px solid #7F8B96; border-radius: 8px; padding: 6px 16px; }
QPushButton:hover { background-color: #EEF2F6; }
QPushButton:pressed { background-color: #E1E7ED; padding-top: 8px; padding-bottom: 6px; }
QPushButton#openFolderButton { background-color: #1FA87A; color: #FFFFFF; border: 1px solid #1A8F68; }
QPushButton#openFolderButton:hover { background-color: #19966B; }
QPushButton#openFolderButton:pressed { background-color: #157E5A; }
QTableWidget, QTextEdit { background-color: #FFFFFF; border: 1px solid #CBD5DF; border-radius: 6px; }
QTableWidget::item:selected { background-color: #E6F2EE; color: #1E2A24; }
QHeaderView::section { background-color: #F2F4F7; border: 1px solid #CBD5DF; padding: 6px 8px; }</string>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="spacing">
        <number>16</number>
     </property>
     <property name="leftMargin">
        <number>0</number>
     </property>
     <property name="topMargin">
        <number>0</number>
     </property>
     <property name="rightMargin">
        <number>0</number>
     </property>
     <property name="bottomMargin">
        <number>0</number>
     </property>
     <item>
        <widget class="QFrame" name="cardFrame">
         <layout class="QVBoxLayout" name="cardLayout">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QLabel" name="titleLabel">
                <property name="styleSheet">
                 <string notr="true">font: 12pt &quot;Microsoft YaHei&quot;; font-weight: 600;</string>
                </property>
                <property name="text">
                 <string>输出管理</string>
                </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="filterRow">
                <property name="spacing">
                 <number>8</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="searchLineEdit">
                    <property name="placeholderText">
                     <string>搜索文件名或任务</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="typeComboBox">
                    <item>
                     <property name="text">
                        <string>全部类型</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                        <string>字幕提取</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                        <string>字幕翻译</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                        <string>烧录压制</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                        <string>视频下载</string>
                     </property>
                    </item>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="statusComboBox">
                    <item>
                     <property name="text">
                        <string>全部状态</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                        <string>已完成</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                        <string>进行中</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                        <string>失败</string>
                     </property>
                    </item>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="fromDateEdit">
                    <property name="calendarPopup">
                     <bool>true</bool>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="toDateEdit">
                    <property name="calendarPopup">
                     <bool>true</bool>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="refreshButton">
                    <property name="text">
                     <string>刷新</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="openFolderButton">
                    <property name="text">
                     <string>打开输出目录</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="exportListButton">
                    <property name="text">
                     <string>导出清单</string>
                    </property>
                 </widget>
                </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="outputsTable">
                <property name="minimumSize">
                 <size>
                    <width>0</width>
                    <height>260</height>
                 </size>
                </property>
                <property name="columnCount">
                 <number>6</number>
                </property>
                <property name="horizontalHeaderLabels">
                 <stringlist>
                    <string>名称</string>
                    <string>类型</string>
                    <string>来源</string>
                    <string>状态</string>
                    <string>时间</string>
                    <string>路径</string>
                 </stringlist>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>140</number>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>110</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="summaryRow">
                <property name="spacing">
                 <number>16</number>
                </property>
                <item>
                 <widget class="QLabel" name="totalCountLabel">
                    <property name="text">
                     <string>总计 0</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="successCountLabel">
                    <property name="text">
                     <string>已完成 0</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="runningCountLabel">
                    <property name="text">
                     <string>进行中 0</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="failedCountLabel">
                    <property name="text">
                     <string>失败 0</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <spacer name="summarySpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                        <width>40</width>
                        <height>20</height>
                     </size>
                    </property>
                 </spacer>
                </item>
             </layout>
            </item>
         </layout>
        </widget>
     </item>
     <item>
        <widget class="QFrame" name="detailsFrame">
         <layout class="QVBoxLayout" name="detailsLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="detailsHeaderRow">
                <property name="spacing">
                 <number>8</number>
                </property>
                <item>
                 <widget class="QLabel" name="detailsTitleLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 11pt &quot;Microsoft YaHei&quot;; font-weight: 600;</string>
                    </property>
                    <property name="text">
                     <string>详情与日志</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <spacer name="detailsHeaderSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                        <width>40</width>
                        <height>20</height>
                     </size>
                    </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="openItemButton">
                    <property name="text">
                     <string>打开文件</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="revealInFolderButton">
                    <property name="text">
                     <string>定位到目录</string>
                    </property>
                 </widget>
                </item>
             </layout>
            </item>
            <item>
             <widget class="QTextEdit" name="detailsTextEdit">
                <property name="minimumSize">
                 <size>
                    <width>0</width>
                    <height>160</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
             </widget>
            </item>
         </layout>
        </widget>
     </item>
    </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
